pkg/utils/utils.go

```diff
@@ -67,10 +67,13 @@ func ExecuteCommand(command string, timeout time.Duration, context *interactive.
 	log.Debugf("Executing command: %s", command)
 
 	values := make(map[string]interface{})
-	// Escapes the double quote char to make a valid json string.
-	values["COMMAND"] = strings.ReplaceAll(command, "\"", "\\\"")
+	// Escapes the double quote and new line chars to make a valid json string for the command to be executed by the handler.
+	values["COMMAND"] = strings.ReplaceAll(command, "\"", `\"`)
+	values["COMMAND"] = strings.ReplaceAll(values["COMMAND"].(string), "\n", `\\n`)
```

Comment:
Should we also escape the other characters at https://www.freeformatter.com/json-escape.html ?

Reply:
Added function to escape chars through marshalling.